eclipse-collections-testutils/src/test/java/org/eclipse/collections/impl/test/VerifyTest.java

```diff
@@ -380,6 +399,413 @@ public void run()
     }
 
     @Test
+    public void assertListMultimapsEquals()
+    {
+        final ListMultimap<Integer, String> multimap1 = FastListMultimap.newMultimap(Tuples.pair(1, "One"), Tuples.pair(2, "Two"), Tuples.pair(2, "TwoTwo"));
+        MutableListMultimap<Integer, String> multimap2 = null;
+        Verify.assertListMultimapsEqual(null, multimap2);
+        multimap2 = FastListMultimap.newMultimap();
+        Verify.assertListMultimapsEqual(FastListMultimap.<Integer, String>newMultimap(), multimap2);
+        multimap2 = FastListMultimap.newMultimap(Tuples.pair(2, "Two"), Tuples.pair(1, "One"), Tuples.pair(2, "TwoTwo"));
+        Verify.assertListMultimapsEqual(multimap1, multimap2);
+
```

Comment:
Some of the test multimaps are declared as new variables, and some reassign existing variables in scope. I'd prefer consistently declaring new variables.